### DISCUSSION TITLE
[Suggestions] Game, replace the word "battle" with "combat", and the word "monster" with "creature" wherever they are used in the game

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#9100] [General]
I suggest replacing the word "battle" with "combat", and the word "monster" with "creature" wherever they are used in the game.
In effect, in the original game, the word "combat" is used for everything concerning a confrontation between a hero and another hero/captain, and between a hero and creatures.
The term "creature" is also used in the original game for everything referring to dwellings, troops to be recruited, etc.
The terms "combat" and "creature" are also used in the original game manual.

In fheroes2:
Here some of the screens concerned (this is not exhaustive):

For combat:
![Combat creature words 001](https://github.com/user-attachments/assets/d38a3153-b8fe-4632-baac-3e9897d8a9bd)
![Combat creature words 002](https://github.com/user-attachments/assets/c0d59f51-d848-4fac-8c91-f26dc8512052)
![Combat creature words 004](https://github.com/user-attachments/assets/00de5884-d597-401f-a7e1-100ecabe0f9c)

For creature:
![Combat creature words 003](https://github.com/user-attachments/assets/74b30c9b-dda1-4deb-8a21-c1a8e050eba4)
![Combat creature words 005](https://github.com/user-attachments/assets/bf4df67c-9e13-4efa-838c-9eae2b1e4319)

**Comment by ihhub** [comment]
I can say one word: **synonyms**. Having variety of words is way better than just using the same word all over the places. Let's use the power of English, not to limit it.

Monster word is used in the original game as well.

**Reply by LeHerosInconnu** [reply]
Hello @ihhub,

> I can say one word: **synonyms**. Having variety of words is way better than just using the same word all over the places. Let's use the power of English, not to limit it.
> 
> Monster word is used in the original game as well.

Although synonyms can be used in everyday language or in texts telling a story, it can become confusing for a new user to use different words to talk about the same thing in an explanatory and referential way, as is the case in a video game.
Older gamers, accustomed to these specific terms, may also be confused by the use of synonyms in place of familiar terms.

**Comment by Mr-Bajs** [comment]
fheroes2 have made some changes from the original game, to make it less confusing and a more standardised terminology. It's pretty fine as it is now. However i could see that replacing troops with units, and monsters with creatures would be an improvement. Synonyms could bring some problems for translators possibly.

fheroes2 
**Army** is now used to refer to the total number of creatures/troops/units a hero or a town have. A hero and a town/castle can only have one army. An hero's army is made up of max 5 units, unless more due to summoning/cloning.
**Unit/troop** is the stack of creatures/monsters a that the player controls on the battlefield. A unit is made up of 1 or more creatures/monsters.
**Creature/monster** is the single individual Titan for example.
